core/dbt/include/global_project/macros/materializations/table/table.sql

```diff
@@ -47,6 +47,8 @@
 
   {{ adapter.rename_relation(intermediate_relation, target_relation) }}
 
+  {% do create_indexes(target_relation) %}
```

Comment:
Should I call `do create_indexes` before or after the `run_hooks(post_hooks, inside_transaction=True)` call?